pywwt/layers.py

```diff
@@ -713,9 +700,7 @@ class TableLayer(HasTraits):
     ).tag(wwt="decay")
 
     selectable = Bool(
-        True,
-        help="Whether sources in the layer "
-        "are selectable (`bool`)"
+        True, help="Whether sources in the layer " "are selectable (`bool`)"
```

Comment:
The autoformatter is a bit conservative here but the adjacent string literals can be combined into a single literal.